hw02/EnemyCollision.cs

```diff
@@ -18,15 +18,23 @@ void Update()
     
     // TODO: Problem 7: Why did I not include the words public or private here?
     // What does it mean when I only write void (Movement)? What does void mean?
+    
+    //public means you can access it globally,while private you can not. if no modifier is provided, it defalts to private
+    //void is a return type, it means the function doesn't return a value.
     void Movement ()
     {
         // TODO: Problem 8: Please explain what the next 4 lines mean.
         // What is Input.GetAxis, transform.Translate, and transform.Rotate? 
         float forwardMovement = Input.GetAxis("Vertical") * speed * Time.deltaTime;
+        //detects forward and backward movement
```

Comment:
Input.GetAxis returns a number between -1 and 1 when you press a key/joystick. Or whenever there is some input. Here, our inputs are `Horizontal` and `Vertical`. Time.deltaTime is the completion time in seconds since the last frame. This helps us to make the game frame-independent.